Update ome-faq.md (#3987)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/ome-faq.md

Not at this time.

> 

Reviewer: Delete this line

> ## Can I send as a shared mailbox and encrypt emails?

Reviewer 2: ```suggestion
## Can I send an encrypted email from a shared mailbox?
```

- We don't support assignment through an email enabled security group. We only support access provided by direct user assignment to the shared mailbox and that automapping is enabled for Exchange Online. Automapping is enabled by default for Exchange Online.

> 

Reviewer: Delete this line

   ```powershell
   Add-MailboxPermission -Identity support@contoso.onmicrosoft.com -User ayla@contoso.com -AccessRights FullAccess -AutoMapping $true
   ```
   

>  ## Can I open encrypted messages sent to another user mailbox with Fullaccess?

Reviewer 2: ```suggestion
 ## Can I open encrypted messages sent to another user's mailbox with Fullaccess?
```

> Users can open encrypted messages as long as they are given direct access and automapping is turned ON. We don't support if the access to granted via an email enabled security group.

Reviewer 2: ```suggestion
Users can open encrypted messages as long as they are given direct access and automapping is turned ON. Access is not allowed if the access is granted via an email-enabled security group.
```
